### ISSUE TITLE
Не отображается на странице.

### ISSUE THREAD

**Issue by AlixanTelnov** [WebTolk/WT-YOOtheme-Yandex-Map#1]
Будьте любезны, не могу настроить отображение вашего модуля.
Может я пришлю вам доступ к сайту и Вы лично посмотрите на причины?

![2024-09-23_15-55-16](https://github.com/user-attachments/assets/950929db-3a97-43da-98c6-169d83d71436)
![photo_2024-09-22_17-10-31](https://github.com/user-attachments/assets/b0c9e6cf-38db-4e26-973f-e85b61d8658b)

**Comment by sergeytolkachyov** [comment]
API ключ вставили в настройках плагина?

**Comment by sergeytolkachyov** [comment]
@gug2 с тебя документация )

**Comment by AlixanTelnov** [comment]
После того как вы о нем напомнили, создал и добавил его. Кстати, если вы делаете карты под YooTheme, полагаю и ключ было бы удобнее добавлять в его интерфейсе настройках модуля. Не видя поля для него и я забыл о его необходимости.
Добавил ключ, карта не появилась.

![2024-09-23_20-08-37](https://github.com/user-attachments/assets/8c829de6-f97b-46fe-99d0-8621db80c22b)

**Comment by AlixanTelnov** [comment]
Господа, можно я скину на info@web-tolk.ru данные для входа на мой проект? Что-бы Вы могли взглянуть живьем.

**Comment by zvencity** [comment]
> Господа, можно я скину на [info@web-tolk.ru](mailto:info@web-tolk.ru) данные для входа на мой проект? Что-бы Вы могли взглянуть живьем.

Удалось решить проблему? А то у меня тоже карты на фронте не видно

**Comment by AlixanTelnov** [comment]
Да, но простите, я не помню как я исправил ситуацию, это было давно)

**Comment by zvencity** [comment]
> Да, но простите, я не помню как я исправил ситуацию, это было давно)

Значит это решаемо, уже хорошо. Если вдруг вспомните, пишите, мало ли это раньше, чем методом тыка найдётся решение. В любом случае спасибо за ответ.

**Comment by sergeytolkachyov** [comment]
@gug2

**Comment by zvencity** [comment]
> [@gug2](https://github.com/gug2)

Спасибо
